core/trino-spi/src/main/java/io/trino/spi/predicate/SortedRangeSet.java

```diff
@@ -1104,17 +1104,11 @@ public Range toRange()
         @Override
         public int compareTo(RangeView that)
         {
-            int compare;
-            compare = compareLowBound(that);
+            int compare = compareLowBound(that);
```

Comment:
now the variable hold only comparison result for the low bound only, so could be renamed

Reply:
How about renaming it to `lowBoundCompare` for readability?

Reply:
Renamed in #13811

Reply:
I don’t think the existing name is less readable. If anything, I’d rename it to “comparison”. The fact that it’s the comparison for the lower bound is pretty obvious, as it’s the only thing being compared. There’s no need to make the name overly verbose — verbosity does not necessarily improve readability.